scalikejdbc-syntax-support-macro/src/test/scala/foo/AutoSpec.scala

```diff
@@ -14,9 +14,10 @@ class AutoSpec extends FlatSpec with Matchers with DBSettings {
     def apply(r: ResultName[Organization])(rs: WrappedResultSet): Organization = autoConstruct(rs, r)
   }
 
-  case class Person(id: Long, name: String, organizationId: Option[Long], organization: Option[Organization] = None, gorupId: Long = 0)
+  case class Person(id: Long, name: String, organizationId: Option[Long], organization: Option[Organization] = None, groupId: Long = 0)
   object Person extends SQLSyntaxSupport[Person] {
     def apply(s: SyntaxProvider[Person])(rs: WrappedResultSet): Person = autoConstruct(rs, s, "organization")
+    override lazy val columns = useField[Person]("organization")
```

Comment:
@gakuzzzz What do you think about this? I still think `entityFields` which returns column names is a unacceptable naming. How about these ones instead? 

```scala
override lazy val columns = autoColumns[Person]("organization")
override lazy val columns = autoColumnsFrom[Person]("organization")
override lazy val columns = entityToColumns[Person]("organization")
```

All of the above names are good to me. Any thoughts?>All